src/platform/utilities/api/service/index.js

```diff
@@ -0,0 +1,103 @@
+const METHODS = {
+  get: 'GET',
+  post: 'POST',
+  delete: 'DELETE',
+  patch: 'PATCH',
+};
+
+const host = '/v0';
+
+export function constructUrl(method, chunks, arg = null) {
+  let url = `${host}/${chunks.join('/')}`;
+  if (typeof arg === 'number') {
+    url += `/${arg}`;
+  } else if (arg != null && typeof arg === 'object' && method === METHODS.get) {
```

Comment:
can you show us an example of how arg is used? is it like 
```
api
.post
.beta_registrations[service]({ query1: "value", query2: "value" })
```

Reply:
With the current setup, arg is used differently for a GET and a POST.  For a GET the arg is encoded as a query string like this:

```js
// methodHandler receives a `get`, so the arg becomes a query string
api.get.appointments({ foo: 'bar', hello: 'world' })
```
https://github.com/department-of-veterans-affairs/vets-website/blob/3ccb87a541ee670131fa3263fcf4f68132cb53ad/src/platform/utilities/api/service/api.unit.spec.js#L50-L71

</br>
For a POST, the arg is treated as json that needs to be stringified and put in the request body.  (We could also have it handle multipart form data if we wanted to).  Here are some examples of that:

The `post` here would result in the invocation treating the same arg as a body object rather than a query string:

```js
// methodHandler receives a `post`, so the arg becomes the body content
api.post.appointments({ foo: 'bar', hello: 'world' })
```

https://github.com/department-of-veterans-affairs/vets-website/blob/3ccb87a541ee670131fa3263fcf4f68132cb53ad/src/platform/utilities/api/service/api.unit.spec.js#L74-L84